specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskStatusRequest.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
I think the format of this file is wrong.  Usually there is some additional metadata in the json structure, with the request body contents nested within a separate "body" property.  You can take a look at one of the examples used for version 2022-02-01-preview to see how it was done previously.

Reply:
Updated